tools/shared/README.md

```diff
@@ -11,122 +11,243 @@ Shared mixins, tools, etc. that support developing Spectrum Web Components.
+
+**When to use:** Use this base class when creating custom elements that need to delegate focus to an internal element (like a button or input) while maintaining proper tabindex management and accessibility.
 
 ```javascript
 import { Focusable } from '@spectrum-web-components/shared';
-import { html } from 'lit-element';
+import { html, TemplateResult } from '@spectrum-web-components/base';
 
 class FocusableButton extends Focusable {
-    public static override get styles(): CSSResultArray {
-        return [...super.styles];
-    }
     public get focusElement(): HTMLElement {
         return this.shadowRoot.querySelector('#button') as HTMLElement;
     }
 
     protected override render(): TemplateResult {
         return html`
-            <button
-                id="button"
-            >
+            <button id="button">
                 Focus for this button is being managed by the focusable base class.
             </button>
         `;
     }
 }
 ```
 
-### FocusVisiblePolyfillMixin
+### Mixins
+
+#### LikeAnchor
 
-Use this mixin if you would like to leverage `:focus-visible` based selectors in your CSS. [Learn more about the polyfill that powers this.](https://www.npmjs.com/package/focus-visible)
+Mix `download`, `label`, `href`, `target`, `rel`, and `referrerpolicy` properties into your element to allow it to act more like an `HTMLAnchorElement`. It also provides a `renderAnchor` method for rendering anchor elements.
 
-### getActiveElement
+**When to use:** Use this mixin when creating custom elements that need to behave like links or buttons with link-like functionality, such as action buttons that can navigate to URLs.
 
-Use this helper to find an `activeElement` in your component. [Learn more about tracking active elements over shadow DOM boundaries.](https://dev.to/open-wc/mind-the-document-activeelement-2o9a)
+```javascript
+import { LikeAnchor } from '@spectrum-web-components/shared';
+import { ReactiveElement, html, TemplateResult } from '@spectrum-web-components/base';
+
+class MyLinkElement extends LikeAnchor(ReactiveElement) {
+    protected render(): TemplateResult {
+        return this.renderAnchor({
+            id: 'my-anchor',
+            className: 'my-link',
+            anchorContent: html`<slot></slot>`,
+        });
+    }
+}
+```
 
-### LikeAnchor
+#### FocusVisiblePolyfillMixin
 
-Mix `download`, `label`, `href`, and `target` properties into your element to allow it to act more like an `HTMLAnchorElement`.
+This mixin coordinates with the focus-visible polyfill to ensure proper behavior across browsers. [Learn more about the polyfill that powers this.](https://www.npmjs.com/package/focus-visible)
```

Comment:
Do we have preferences on including the period in the linked text? I found the content standards for links, but to me, [the examples](https://spectrum.adobe.com/page/link/#Content-standards) aren't explicit enough about including the punctuation, although some of the screenshots seem to suggest _not_ including the period in the link. 

My default is not to include them usually, but I can't actually tell you why.